### PR TITLE
add some test coverage for `genome-env-next`

### PR DESCRIPTION
**Pull request by nnutter**
There is more to be done but this is first pass at adding some sort of testing
for `genome-env-next`.

### CODE REVIEW

**Comment by nnutter** [comment]
```
$ git checkout origin/pull/494/merge 
...
HEAD is now at 3722cf9... Merge 182b9718dde21ded89b8127270bd65ce4e14d9f2 into 53a549fbe8e6b783d1ff1951fb31e744e82d6594
$ git status -s
$ cd genome-env-tests/
$ bats *.bats
 ✓ basic test: genome-env-next -D
 ✓ basic test: genome-env-next -U
 ✓ basic test: genome-env-next -u
 ✓ basic test: genome-env-next -W
 ✓ basic test: genome-env-next -w
 ✓ unclean submodule fails
 ✓ submodule with commits fails

7 tests, 0 failures
```

File: bin/genome-env-next [hunk 1]
```diff
@@ -429,43 +433,76 @@ function setup_env {
...
+    fi
+
+    if test -z "$DISABLE_UR"
+    then
+        export PERL5LIB=$UR_REPO/lib:$PERL5LIB
+
+        if ! genome-perl -MUR -e 1 2> /dev/null
         then
-            fatal $BUILD_ERR "failed to load $MODULE"
+            genome-perl -MUR -e 1
```
Comment (tmooney) [inline bin/genome-env-next]: What is this second `genome-perl` without the `2> /dev/null` for?

In general, would it be worthwhile to extract `genome-perl -M$MODULE -e 1 2> /dev/null` into a function?

Comment (nnutter) [inline bin/genome-env-next]: A leftover debug statement.  I'll remove it.

**Comment by tmooney** [comment]
+1, I'm excited to have a testing framework for our bash stuff!  :basketball:

**Comment by nnutter** [comment]
```
$ git checkout origin/pull/494/merge
...
HEAD is now at 4d25e8d... Merge a465294d5998f5128af91b2c2371b9f4a8ec0b55 into 44e5f82c3f502db0d0541507a0827f07f494178a
$ cd genome-env-tests/
$ git status -s
$ bats *.bats
 ✓ basic test: genome-env-next -D
 ✓ basic test: genome-env-next -U
 ✓ basic test: genome-env-next -u
 ✓ basic test: genome-env-next -W
 ✓ basic test: genome-env-next -w
 ✓ unclean submodule fails
 ✓ submodule with commits fails

7 tests, 0 failures
```

**Pull request merged**